jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -229,42 +226,16 @@ class MachineView extends React.Component {
         toggleScaleUp={this._toggleScaleUp.bind(this)} />);
   }
 
-  /**
-    Scroll a column to a machine or container.
-    @param {String} id The machine id to scroll to.
-    @param {Boolean} isContainer Whether the machine is a container.
-  */
-  _scrollToMachine(id, isContainer=false) {
-    // Get the correct column that contains the machine to scroll to.
-    const column = ReactDOM.findDOMNode(isContainer ?
-      this.refs.containersColumn : this.refs.machinesColumn);
-    // Get the content node that scrolls.
-    const content = column.querySelector('.machine-view__column-content');
-    const machine = ReactDOM.findDOMNode(
-      this.refs[`${isContainer ? 'container' : 'machine'}-${id}`]);
-    if (content && machine) {
-      content.scrollTop = machine.offsetTop - content.offsetTop;
-    }
-  }
-
   /**
     Handle selecting a machine.
 
     @param {String} id The machine id to select. If the id is null, no machine
       is selected.
-    @param {Boolean} scrollToMachine Whether to scroll to the selected machine.
   */
-  selectMachine(id, scrollToMachine=true) {
-    if (id === null) {
+  selectMachine(id) {
+    if (!id || id === null) {
```

Comment:
null evals to false so the second part of the OR is redundant.

Reply:
You are correct!